src/utils/fileSystem.js

```diff
@@ -0,0 +1,696 @@
+/* eslint-disable global-require */
```

Comment:
You need to think more critically about if you need to bring over files like this. In this case, you definitely do not!

The only method from this file that you are actually using is `writeFile`, which as you can see below is a simply a wrapper around `fs.writeFile`. Why didn't you use this directly? `fs` is a core node module, and is included in Electron automatically. 

By importing this whole file, you duplicated many existing filesystem methods that Architect already has, which creates confusion and introduces issues with maintainability. Another issue is that this file is very large, and all the unused code you imported is simply wasted space.